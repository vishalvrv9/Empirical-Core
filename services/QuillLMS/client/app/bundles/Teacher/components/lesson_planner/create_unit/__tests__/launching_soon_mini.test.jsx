import React from 'react';
import { shallow } from 'enzyme';

import LaunchingSoonMini from '../launching_soon_mini';

describe('LaunchingSoonMini component', () => {

  it('should render', () => {
    expect(shallow(<LaunchingSoonMini />)).toMatchSnapshot();
  });

});
