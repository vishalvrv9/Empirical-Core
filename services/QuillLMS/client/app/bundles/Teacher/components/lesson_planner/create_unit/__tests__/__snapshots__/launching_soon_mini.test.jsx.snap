// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaunchingSoonMini component should render 1`] = `
<div
  className="launching-soon-mini"
>
  <h3>
    Launching Soon
  </h3>
  <div
    className="titles"
  >
    <p>
      Post Test Diagnostic
    </p>
    <p>
      Elementary Level Diagnostic
    </p>
  </div>
  <p>
    We are launching a post test diagnostic and an elementary level diagnostic next year. Stay tuned!
  </p>
</div>
`;
